src/main/java/rx/observables/AbstractOnSubscribe.java

```diff
@@ -98,7 +98,7 @@
  * <pre><code>
  * Iterable<T> iterable = ...;
  * AbstractOnSubscribe.create(s -> {
- *   Iterator<T> it = s.state();
+ *   Iterator<T> it = iterable.iterator();
```

Comment:
There is no typo here, the per-subscriber state of iterator is created on line 108.